Makefile

```
@@ -215,3 +215,7 @@ scan-licenses:
 .PHONY: generate
 generate:
 	$(MAKE) for-all CMD="go generate ./..."
+
+.PHONY: create-plugin-docs
+create-plugin-docs:
+	cd cmd/plugindocgen; go run *.go
```

This is super nitpicky and I don't even know if there's an actual difference so feel free to ignore this, but usually I see this:
```suggestion
	cd cmd/plugindocgen; go run .
```